x-pack/platform/packages/shared/kbn-classic-stream-flyout/src/components/create_classic_stream_flyout/create_classic_stream_flyout.tsx

```diff
@@ -78,6 +152,23 @@ export const CreateClassicStreamFlyout = ({
   const goToNextStep = () => setCurrentStep(ClassicStreamStep.NAME_AND_CONFIRM);
   const goToPreviousStep = () => setCurrentStep(ClassicStreamStep.SELECT_TEMPLATE);
 
+  const handleCreate = useCallback(async () => {
+    setHasAttemptedSubmit(true);
+    setIsValidating(true);
+
+    try {
+      const result = await validateStreamName(streamName, onValidate);
```

Comment:
if this fails, we are silently swallowing the error since we dont have a catch. Is this something we should worry about?